rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -86,11 +158,21 @@ def test_paraswap_swap_with_fee(accountant: 'Accountant'):
             free_amount=ZERO,
             taxable_amount=FVal(swap_amount_str),
             price=Price(ONE),
-            pnl=PNL(taxable=FVal(swap_amount_str), free=ZERO),
-            cost_basis=CostBasisInfo(taxable_amount=FVal(swap_amount_str), taxable_bought_cost=ZERO, taxfree_bought_cost=ZERO, matched_acquisitions=[], is_complete=False),  # noqa: E501
-            index=0,
+            pnl=PNL(taxable=FVal(swap_amount_str) / 2, free=ZERO),
```

Comment:
this `/ 2` comes from the price right? Maybe create a variable that points `MOCKED_PRICES[event.asset][event.timestamp]`